exporter/jaeger/jaegerthrifthttpexporter/config.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jaegerthrifthttpexporter
+
+import (
+	"time"
+
+	"github.com/open-telemetry/opentelemetry-service/config/configmodels"
+)
+
+// Config defines configuration for Jaeger Thrift over HTTP exporter.
+type Config struct {
+	configmodels.ExporterSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
+
+	// URL is the URL to send the Jaeger trace data to (e.g.:
+	// http://some.url:14268/api/traces).
+	URL string `mapstructure:"url"`
```

Comment:
Nit: can we use `endpoint` to be consistent with other configs?

Reply:
We have been consistent on endpoint for receivers because they should always take the same syntax (host:port) but for exporters we had a different conversation in one previous PR: https://github.com/open-telemetry/opentelemetry-service/pull/207#discussion_r309012866 let's know if you agree with that.

Reply:
SGTM